src/status_im/contexts/wallet/send/select_address/tabs/view.cljs

```diff
@@ -27,15 +27,30 @@
                                                     :stack-id  :screen/wallet.select-address}])}]))
             other-accounts))))
 
+(defn- recent-transactions
+  [theme]
+  (let [recent-recipients (rf/sub [:wallet/recent-recipients])]
+    (if (zero? (count recent-recipients))
```

Comment:
Could be `(empty? recent-recipients)` ?

Reply:
thanks for the suggestion, will update this.